dali/operators/generic/constant.cc

```diff
@@ -25,41 +25,48 @@ namespace dali {
 DALI_SCHEMA(Constant)
   .DocStr(R"code(Produces a batch of constant tensors.
 
-The floating point input data should be placed in `fdata` argument and integer data in `idata`.
-The data is a flat vector of values or a single scalar. The data is then reshaped according
-to the `shape` argument. If the data is scalar, it will be broadcast to fill the entire shape.
+In ``idata``, the floating point input data should be placed in the ``fdata`` argument
+and integer data. The data, which is a flat vector of values or a scalar, is then reshaped
+according to the shape argument. If the data is scalar, it will be broadcast
+to fill the entire shape.
```

Comment:
```suggestion
The floating point input data should be placed in the ``fdata`` argument
and integer data in ``idata``. The data, which is a flat vector of values or a scalar, is then reshaped
according to the ``shape`` argument. If the data is scalar, it will be broadcast
to fill the entire shape.
```

Reply:
Done